src/hf_hydrodata/data_catalog.py

```diff
@@ -633,5 +640,5 @@ def test_get_tables():
     """Test get_table_names."""
 
     hf_hydrodata.gridded.HYDRODATA = "/hydrodata"
-    table_names = hf.get_table_names()
+    table_names = hf_hydrodata.get_table_names()
```

Comment:
What's up with this file? Should this be in a `tests` directory, and why does it use `hf_hydrodata` instead of the import alias of `hf` that the rest of the file uses?